preview/Win7Msix/Win7MSIXInstaller/AddRemovePrograms.cpp

```diff
@@ -13,6 +13,11 @@ const PCWSTR AddRemovePrograms::HandlerName = L"AddRemovePrograms";
 
 HRESULT AddRemovePrograms::ExecuteForAddRequest()
 {
+    if (m_msixRequest->GetIsInstallCancelled())
+    {
+        return ERROR_INSTALL_USEREXIT;

```

Comment:
>ERROR_INSTALL_USEREXIT [](start = 15, length = 22)

this is win32, you need to HRESULT_FROM_WIN32 to return an HRESULT.

Reply:
Updated to return HRESULT_FROM_WIN32